dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/bootstrap/DubboBootstrap.java

```diff
@@ -1067,19 +1082,21 @@ private void unexportMetadataService() {
     private void exportServices() {
         configManager.getServices().forEach(sc -> {
             // TODO, compatible with ServiceConfig.export()
-            ServiceConfig serviceConfig = (ServiceConfig) sc;
+            ServiceConfig<?> serviceConfig = (ServiceConfig<?>) sc;
             serviceConfig.setBootstrap(this);
 
-            if (exportAsync) {
```

Comment:
Has the definition of exportAsync cleaned up in Bootstrap?

Reply:
Already annotated with @Deprecated, But It's better to remove the definition, I'll change it